### ISSUE TITLE
Arms action list improvement.

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#820]
Originally reported on Google Code with ID 821

```
This is just a copy paste from the feedback section. I've confirmed it with my own warrior
I imported from battle.net.

Two improvements to Arms warrior priority list:

1) Golemblood potion should be aligned with second recklessness, i.e. change actions+=/golemblood_potion,if=!in_combat|buff.bloodlust.react
into actions+=/golemblood_potion,if=!in_combat|buff.recklessness.react|target.time_to_die<26

2) It is a dps loss to slam too much when colossus smash is up, at least with T12 4pc
bonus. Present priority list has actions+=/colossus_smash,if=buff.colossus_smash.remains<0.5
actions+=/execute,if=(buff.deadly_calm.up|buff.recklessness.up) actions+=/mortal_strike
actions+=/overpower actions+=/execute actions+=/slam,if=(cooldown.mortal_strike.remains>=1.5&(rage>=35|buff.deadly_calm.up|buff.colossus_smash.up))|(cooldown.mortal_strike.remains>=1.2&buff.colossus_smash.remains>0.5&rage>=35)

This should be modified to actions+=/colossus_smash,if=!buff.colossus_smash.up actions+=/execute,if=(buff.deadly_calm.up|buff.recklessness.up)
actions+=/mortal_strike actions+=/overpower actions+=/execute actions+=/colossus_smash,if=buff.colossus_smash.remains<=1.5
actions+=/slam,if=(cooldown.mortal_strike.remains>=1.5&(rage>=35|buff.deadly_calm.up|buff.colossus_smash.up))|(cooldown.mortal_strike.remains>=1.2&buff.colossus_smash.remains>0.5&rage>=35)

Basically Mortal Strike, Overpower and Execute are allowed to drop Colossus Smash buff,
while Slam is not. Increase in numbers of melee strikes, deep wounds, opportunity strikes
and fiery attacks outweight the Slam dmg (because of swing timer pausing).

With 420-4 and 50k iterations unmodified T12N 28700 dps, T12H 32545 dps. Modified T12N
28923 dps, T12H 32844.5 dps.


What version of the product are you using? On what operating system?
r9101 Win7 x64

Please provide any additional information below.

My own warrior saw gains of ~30 dps with better golemblood usage, and another 200 with
the change to push more colossal smashes.
```

Reported by `arexdramon` on 2011-08-11 05:24:51
